src/components/common/EditableInput.vue

```diff
@@ -41,6 +48,10 @@ export default {
     }
   },
   methods: {
+    onCopy() {
+      CopyToClipboard(this.value)
```

Comment:
Import Copy to clipboard here so it's loaded when button clicked, not everyone will copy the color.